src/alfasim_sdk/default_tasks.py

```diff
@@ -3,16 +3,49 @@
 import subprocess
 import sys
 from pathlib import Path
+from typing import Any
 from typing import Union
 from zipfile import ZipFile
 
 from colorama import Fore
 from colorama import Style
 from hookman.hookman_generator import HookManGenerator
+from invoke import Collection
 from invoke import Exit
+from invoke import Task
 from invoke import task
 from strictyaml.ruamel import YAML
 
+sdk = Collection()
```

Comment:
I've called the collections `sdk` instead of `sdk_tasks`, because when we call then we will use `sdk.tasks['example']` instead of `sdk_tasks.tasks['example']`, which seemed a little bit of repetition.

What do you guys think?

Reply:
Not sure, I find the symbol `sdk`, in a project called `alfasim-sdk`, to be a *collection of sdk tasks* to be a bit surprising... 

But OK, not sure if this will be a problem either, go for it.

Reply:
I agree, but I cannot think of a better name. Perhabs `sdk_collection` or `sdk_namespace` (`sdk_ns` for short) but it felt strange as well.

Reply:
`sdk_ns` I like